modules/node_modules/@ncigdc/components/DownloadButton.js

```diff
@@ -28,14 +31,20 @@ const DownloadButton = ({
   filters = {},
   active,
   setActive,
+  altMessage = false,
+  style = {},
   extraParams = {},
+  dispatch,
 }) => {
-  const icon = <i key="icon" className={`fa fa-download ${active ? 'fa-spinner fa-pulse' : ''}`} />;
+  const icon = active ? <Spinner /> : <DownloadIcon />;
```

Comment:
If you are suggesting icons are only for sighted people to show the purpose of the button then I think I disagree. I have seen multiple sites that use icons as action buttons and in those cases doing something like this would not be wrong.

`<i class="fa fa-undo" aria-label="Undo" onClick="undo()"></i>`

In this particular case however I am suggesting adding 'aria-label' on the button and not on the icon of course. With that we don't have to create a new component `Hidden` which I think is unnecessary when we have HTML core attribute to help us with accessibility.